torch/csrc/jit/operator_upgraders/README.md

```diff
@@ -0,0 +1,95 @@
+# Guidance for Operator Developer
+
+After you change either the operator signature in a BC breaking way or the semantics of the operator, you will need to write an “upgrader” to make the change non-BC breaking iff they are used in TorchScript or mobile. In general, you can know your operator is BC breaking, if it fails `test/forward_backward_compatibility/check_forward_backward_compatibility.py `
+
+The steps to write upgrader:
+
+1. Prepare a model before making changes to the operator, following the process:
+     1. add a module in `test/jit/fixtures_srcs/fixtures_src.py`. The reason why switching to commit is that, an old model with the old operator before the change is needed to ensure the upgrader is working as expected. In `test/jit/fixtures_srcs/generate_models.py`, add the module and its corresponding changed operator like following
```

Comment:
what does this mean?

Reply:
Add more explanation and hope it's clearer.